docs/Account/methods.md

```diff
@@ -0,0 +1,284 @@
+---
+title: Methods
+---
+
+The SDK provides the following methods for a smart account.
+
+## BiconomySmartAccountV2
+### Create
+
+The `create()` method is used to create an instance of the Biconomy Smart Account V2. This method takes a configuration object and returns the smart account instance.
+
+The addresses of ERC-4337 smart accounts follow a deterministic pattern. This enables you to get the address off-chain before the account is actually deployed.
+
+**Usage**
+
+```jsx
+let provider = new ethers.providers.JsonRpcProvider("rpcUrl");
+let signer = new ethers.Wallet("private key", provider);
+
+
+const smartAccount = await BiconomySmartAccountV2.create({
+  signer: signer,
+  chainId: ChainId.POLYGON_MUMBAI, // Specify the desired chain (e.g., Polygon Mumbai)
+  bundlerUrl: "", // bundler URL (required)
+  biconomyPaymasterApiKey: "", // Instance of the paymaster
+});
+
+```
+
+**Parameters**
+
+*required params are explicitly mentioned*
+
+- config (`object`, required): A `BiconomySmartAccountV2Config` object containing configuration options for creating the smart account.
+  - signer(`Signer`, required): The signer instance that will need to be passed. 
```

Comment:
I don't think signer is required. 
If this includes 3.1.2 then @VGabriel45 can help you with conditional props and overall content on what is required and reason behind that. 

like... if defaultValidationModule is not provided we will attach instance of ECDSAValidationModule created using provided EOA signer.

Reply:
<img width="1033" alt="Screenshot 2024-01-04 at 20 09 56" src="https://github.com/bcnmy/docs/assets/56271768/55200632-415f-402d-adc2-3f413d4040a4">

Here is how the new types should be. signer is required as long as defaultValidationModule is not passed, it is either one or another.